.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,9 @@
 - [ ] commit messages are meaningful and follow good commit message guidelines
 - [ ] README and other documentation has been updated / added (if needed)
 - [ ] tests have been updated / new tests has been added (if needed)
+- [ ] Screenshots are attached for UI change (if needed)
```

Comment:
this should not be added here but should be updated in this template https://github.com/hmcts/civil-general-apps-ccd-definition/blob/master/.github/PULL_REQUEST_TEMPLATE.md since ga ccd deals with mainly ui changes and this is just orchestration microservice